src/eva/data/ioda_obs_space.py

```diff
@@ -22,13 +22,13 @@
 
 def subset_channels(ds, channels, add_channels_variable=False):
 
-    if 'channel' in list(ds.dims):
+    if 'Channel' in list(ds.dims):
```

Comment:
I don't know why this was lower case before, but I think it still has to be, as the CI tests fail now for not being able to find 'channel'

Reply:
@CoryMartin-NOAA I am a little confused by that warning message as I thought this referred to the dimensions read in to the data space.  [ioda_obs_space.amsua_n19.hofx.2020-12-14T210000Z](https://github.com/JCSDA-internal/eva/blob/develop/src/eva/tests/data/ioda_obs_space.amsua_n19.hofx.2020-12-14T210000Z.nc4) is using upper case `Channel`.  Also that warning message shows up when running the current version.

Reply:
this isn't a warning though, it's failing:
```
KeyError: "'channel' is not a valid dimension or coordinate"
```